lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -136,7 +136,10 @@ class SideTitles with EquatableMixin {
   /// We recommend you to use [SideTitleWidget].
   final GetTitleWidgetFunction getTitlesWidget;
 
-  /// It determines the maximum space that your titles need,
+  /// It determines the maximum width that your titles need
+  final double? reservedWidth;
```

Comment:
Along with reservedWidth, can we have a `reservedHeight` propety as well? Will be useful for widgets on y-axis

Reply:
I renamed reservedSize to reservedHeight and stretch the titles in the main axis direction.